### ISSUE TITLE
LibRealSense works on Linux, why not in Open3D

### ISSUE THREAD

**Issue by rafael2k** [IntelVCL/Open3D#469]
According to librealsense upstream, Linux is supported:
https://github.com/IntelRealSense/librealsense

But Open3D does not compile with system installed librealsense, nor with embedded one (indeed, if compiled with embedded one tweaking the build system, I get no devices found). Indeed, for Linux, it's explicitly disabled...

**Comment by qianyizh** [comment]
The current RealSense support in Open3D is a legacy one I did two (?) years ago. It is legacy. Back then I only tested the R200 sensor. The library was not well-supported by Linux. Some patches have to be made to the kernel and some drivers have to be installed. There were also some minor issues (e.g., can't lock exposure of the color camera, etc) when used on Linux. So I made the decision to turn it off. After that attempt, the RealSense interface has not been changed for years.

And yes, Open3D should support the latest RealSense camera, as well as the latest librealsense. We have a plan, but have got enough time to look into it. The issue is tracked in #231
